orderer/common/cluster/deliver_test.go

```diff
@@ -187,7 +185,7 @@ func (ds *deliverServer) Deliver(stream orderer.AtomicBroadcast_DeliverServer) e
 
 	select {
 	case <-timer.C:
-		panic("timed out waiting for seek assertions to receive a value")
+		ds.t.Fatalf("timed out waiting for seek assertions to receive a value\n")
```

Comment:
why the newline here and not on the one at line 205?

Reply:
good catch, done